docs/de/docs/tutorial/first-steps.md

```diff
@@ -0,0 +1,333 @@
+
+Es gibt Dutzende Alternativen, alle basierend auf OpenAPI. Sie können jede dieser Alternativen problemlos zu Ihrer Anwendung hinzufügen, die Sie mit **FastAPI** erstellt haben.
+
+Ebenfalls können Sie es verwenden, um automatisch Code zu generieren, für Clients, die mit Ihrer API kommunizieren. Zum Beispiel für Frontend-, Mobil- oder IoT-Anwendungen.
+
+## Rekapitulation, Schritt für Schritt
+
+### Schritt 1: Importieren von `FastAPI`
+
+```Python hl_lines="1"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+`FastAPI` ist eine Python-Klasse, die alle Funktionalität für Ihre API bereitstellt.
+
+!!! Hinweis "Technische Details"
+    `FastAPI`  ist eine Klasse, die direkt von `Starlette` erbt.
+
+    Sie können alle <a href="https://www.starlette.io/" class="external-link" target="_blank">Starlette</a> Funktionalitäten auch mit `FastAPI` nutzen.
+
+### Schritt 2: Erzeugen einer `FastAPI` "Instanz"
+
+```Python hl_lines="3"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Hier ist die `app`-Variable eine "Instanz" der Klasse `FastAPI`.
+
+Dies wird der Einstiegspunkt jeder Interaktion sein, um ihre ganze API zu erstellen.
+
+Diese `app`  ist die gleiche, auf die sich `uvicorn` im Kommando bezieht:
+
+<div class="termy">
+
+```console
+$ uvicorn main:app --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+Wenn Sie Ihre Anwendung wie folgt erstellen:
+
+```Python hl_lines="3"
+{!../../../docs_src/first_steps/tutorial002.py!}
+```
+
+Und das in eine Datei `main.py` einfügen, dann würden Sie `uvicorn` wie folgt aufrufen:
+
+<div class="termy">
+
+```console
+$ uvicorn main:my_awesome_api --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+### Schritt 3: Erstellen einer *Pfad-Operation*
+
+#### Pfad
+
+"Pfad" bezieht sich hier auf den letzten Teil der URL, beginnend mit dem ersten `/`.
+
+Sprich, in einer URL wie:
+
+```
+https://example.com/items/foo
+```
+
+... wäre der Pfad:
+
+```
+/items/foo
+```
+
+!!! Info
+    Ein "Pfad" wird häufig auch als "Endpunkt" oder "Route" bezeichnet.
+
+Bei der Erstellung einer API ist der "Pfad" die Hauptmethode zur Trennung von "Anliegen" und "Ressourcen".
+
+#### Operation
+
+"Operation" bezieht sich hier auf eine der HTTP-"Methoden".
+
+Eine von diesen:
+
+* `POST`
+* `GET`
+* `PUT`
+* `DELETE`
+
+... und die etwas Exotischeren:
+
+* `OPTIONS`
+* `HEAD`
+* `PATCH`
+* `TRACE`
+
+Im HTTP-Protokoll können Sie mit jedem Pfad über eine (oder mehrere) dieser "Methoden" kommunizieren.
+
+---
+
+Wenn Sie APIs bauen, verwenden Sie normalerweise diese spezifischen HTTP-Methoden, um eine bestimmte Aktion durchzuführen.
+
+Normalerweise verwenden Sie:
+
+* `POST`: um Daten zu erzeugen.
+* `GET`: um Daten zu lesen.
+* `PUT`: um Daten zu aktualisieren.
+* `DELETE`: um Daten zu löschen.
```

Comment:
Hmm, im Original werden hier die Worte "create, read, update, delete" verwendet, was eine Referenz auf CRUD ist, das geht in der Übersetzung verloren. Vielleicht sollte man das in Klammern dahinter schreiben, z.B. "... um Daten zu erzeugen (create)"?

Reply:
Guter Vorschlag, committed